lib/bolt/inventory/group.rb

```diff
@@ -32,6 +32,14 @@ def initialize(data)
         @rest = data.reject { |k, _| %w[name nodes config groups].include? k }
       end
 
+      def check_deprecated_config(context, name, config)
```

Comment:
Added validation that `config: transports` isn't used @adreyer @katelopresti 

Curious what you think about having Bolt link directly to the docs site.

Reply:
+1 I found this helpful